### DISCUSSION TITLE
Upscaling reduces resemblance. Is there a way around?

### DISCUSSION THREAD

**Discussion by thinkingkoala** [lllyasviel/Fooocus#2569] [Q&A]
I've found a perfect method to create an AI clone of myself and the initial images look amazing
But when I upscale them they seem to lose the resemblance and vary quite a bit from the original images.

Is there a way to preserve the same facial features of my character and only increase the resolution?
I already use the same seed, the same prompt, and enabled _"Mixing Image Prompt and Vary/Upscale"_
Am I missing something?

Here are the initial and the upscaled version. As you can see some features were changed during the upscale process
<img src="https://github.com/lllyasviel/Fooocus/assets/139218006/e3d48a14-966e-4d1a-aff8-ca07707680c6" width="300">
<img src="https://github.com/lllyasviel/Fooocus/assets/139218006/89244e2a-c7d6-4fb4-b8a4-d9c423f1ce9f" width="300">

**Comment by ILikeToasters** [comment]
What are your settings for faceswap when upscaling? The fast upscale will not really change the image but also will not really add detail. The other two upscales do change the image but usually faceswap will work well with that in my experience. I use a weight of at least .9 on the faceswap.

**Reply by thinkingkoala** [reply]
It's not a face swap but LoRa and embedding

**Reply by ILikeToasters** [reply]
Okay. I saw "Mixing Image Prompt and Vary/Upscale" and figured that was what you were talking about.

**Reply by bsdesigner** [reply]
The image will not change when enlarged if you specify the same seed as in the original image
